makeDockerfile.ts

```diff
@@ -99,7 +99,7 @@ WORKDIR /usr/src/upgrade-test-scripts
 COPY --link ./upgrade-test-scripts/install_deps.sh /usr/src/upgrade-test-scripts/
 RUN --mount=type=cache,target=/root/.yarn ./install_deps.sh ${proposalIdentifier}:${proposalName}
 
-COPY --link --chmod=755 ./upgrade-test-scripts/run_eval.sh /usr/src/upgrade-test-scripts/run_eval.sh
+COPY --link --chmod=755 ./upgrade-test-scripts/run_eval.sh /usr/src/upgrade-test-scripts/
```

Comment:
I'm curious... is this just conciseness? or is there a difference in behavior?

Reply:
just consistency with the other copies which copy multiple files (and therefore have to specify a dir target instead of filename)